luxonis_ml/data/datasets/luxonis_dataset.py

```diff
@@ -404,15 +413,22 @@ def get_classes(
 
         return classes, classes_by_task
 
-    def get_skeletons(self) -> Dict[str, Dict]:
+    def get_skeletons(self, sync_mode: bool = False) -> Dict[str, Dict]:
         """Returns the dictionary defining the semantic skeleton for each class using
         keypoints.
 
         @rtype: Dict[str, Dict]
         @return: A dictionary mapping classes to their skeleton definitions.
         """
+        if sync_mode:
+            local_file = osp.join(self.metadata_path, "skeletons.json")
+            self.fs.get_file("metadata/skeletons.json", local_file)
+            with open(local_file) as file:
+                skeletons = json.load(file)
+        else:
+            skeletons = self.datasets[self.dataset_name]["skeletons"]
```

Comment:
Could this be empty?
Is it valid load if it's empty or should we also return a warning?

Reply:
@tersekmatija I've issued a warning here. Could you please check this fix?